packages/react-native-reanimated/src/updateProps/updateProps.ts

```diff
@@ -62,6 +63,9 @@ if (SHOULD_BE_USE_WEB) {
     if ('transformOrigin' in updates) {
       updates.transformOrigin = processTransformOrigin(updates.transformOrigin);
     }
+    if('transform' in updates){
```

Comment:
```suggestion
    if ('transform' in updates) {
```

You can merge this suggestion or run `yarn format` in the `packages/react-native-reanimated` dir to fix the linting error on CI.